src/eos_parse.c

```diff
@@ -207,7 +210,7 @@ void parsePermissionField(uint8_t *in, uint32_t inLength, const char fieldName[]
     uint32_t accountWrittenLength = 0;
     
     parseNameField(in, inLength, fieldName, arg, read, &accountWrittenLength);
-    strcat(arg->data, "@");
+    strlcat(arg->data, "@", 1);
```

Comment:
should be `strlcat(arg->data, "@", sizeof(arg->data));`

Reply:
thx, done